tests/openassetio/test/managerValidator/fixtures.py

```diff
@@ -0,0 +1,14 @@
+"""
```

Comment:
[fix] Can we rename this file to make it clearer that it is a sample fixtures file for one of the test cases, not fixtures for the test cases them selves and move it to a `resources` folder, or have one created on the fly in the tmpdir by the test? This would also avoid the potential confusion the comment attempts to avoid.

Reply:
Great idea, used `tempfile` instead in 9a3408c